caPutLogApp/caPutJsonLogTask.cpp

```diff
@@ -11,7 +11,11 @@
  * - Initial version.
 */
 
+// Windows needs this macro to avoid confict with min/max macros
+#define NOMINMAX
+
 // Standard library imports
+#include <algorithm>  // Windows needs this for std::min/max
```

Comment:
This comment is a little misleading.  [`std::min()`](https://en.cppreference.com/w/cpp/algorithm/min) is defined in `<algorithm>` for all targets.

Reply:
But the other targets don't need `<algorithm>`.

Reply:
I can remove the comment. Just wanted to avoid that someone deletes it because "it is not needed".

Reply:
That this header may sometimes be implicitly included by others is incidental.  Standard C++ clearly defines the association between `std::min()` and `<algorithm>`.

imo. the best guard against regression would be to add CI tests.

Reply:
Feel free to add some. :-)

Reply:
Where is the code that requires `std::min()` when built for Windows?

There are macros for `min()` and `max()` defined in one of the source files inside a `#ifndef` test; maybe they should be replaced instead?

Reply:
The code lines are in `CaPutJsonLogTask::calculateMin` and `CaPutJsonLogTask::calculateMax` starting from line 594 to line 659.

Reply:
I have remove the comment.

Reply:
> Feel free to add some. :-)

I make a habit of adding test builds with [ci-scripts](https://github.com/epics-base/ci-scripts) when I have reason to touch a support module which doesn't have any.  I would encourage others to do the same.